schema/resolvers.ts

```diff
@@ -78,6 +78,23 @@ export const resolvers = {
         .populate('portfolio');
       return response;
     },
+
+    async application(
```

Comment:
I see this already present before. So, what's new in here? Maybe I am missing something.